src/components/Dashboard/Beneficiary/Beneficiary.container.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from "react-redux";
+import { getAddress } from "../../../modules/contract/selectors";
+import Beneficiary from "./Beneficiary";
+
+export const mapState = (state) => {
+  return {
+    address: getAddress(state),
+  };
+};
+
+export const mapDispatch = (dispatch) => ({});
+
+export default connect(mapState, mapDispatch)(Beneficiary);
```

Comment:
Are you using mapDispatch in other place? You can set mapDispatch as null if it's not necessary or remove it from the arguments.
`export default connect(mapState)(Beneficiary);`